nss-day-cohort-48/rare-api-orchestra-outlaws
# Admin Fixtures and Post Tag and Reaction properties

#### Changes Made

1. Modified `admin.py` to register models to admin site
2. Changed User 2 fixture to isStaff: true
3. Added token fixture for User 2
4. Add User 3 superuser fixture
5. Modified `models/post.py' to include properties for displaying the tags and reactions associated with a post
6. Modified `views/post.py` PostSerializer to return tags and reactions properties on a Post

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout mec-posts2
   ```
2. rebuild database and runserver
3.Go to localhost:8000/admin
4. sign in as username: 'superuser' password: 'password'
5. user should be able to view all of the rare_api tables
6. GET /posts
7. posts should have tags and reactions properties that display a list of tags and reactions associated with their associated posts
